src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -228,17 +219,17 @@ private IServiceCollection BuildHostingServices()
             return services;
         }
 
-        private string ResolveApplicationBasePath(string applicationBasePath, string basePath)
+        private string ResolveContentRootPath(string contentRootPath, string basePath)
         {
-            if (string.IsNullOrEmpty(applicationBasePath))
+            if (string.IsNullOrEmpty(contentRootPath))
```

Comment:
Yes the `contentRootPath` can be set as an environment variable. By default it points to the `ApplicationBasePath` of the default `IApplicationEnvironment`